content/blog/syntax-highlighting-with-tina.md

```diff
@@ -0,0 +1,192 @@
+---
+title: 'How to Use Syntax Highlighting with TinaMarkdown'
+date: '2022-01-20T00:00:00-04:00'
+author: James Perkins
+last_edited: '2022-01-20TT00:00:00Z'
+opengraph:
+  image:
+    url: >-
+      https://res.cloudinary.com/forestry-demo/image/upload/v1642688324/blog-media/syntax-highlighting-with-tina/syntax-highlight-cover.png
+    width: '1920,'
+    height: '1080,'
+    alt: 'Syntax Highlighting with Tina'
+---
+
+👋 Hello and welcome. Today we are going to walk you through how to use syntax highlighting with TinaMarkdown. Through a very basic setup and example we will walk you through adding syntax highlighting so that your editors will love you (or at least highly appreciate you).
+
+Prefer to learn through videos? Check out our video covering syntax highlighting:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Qp-7YlGisBM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="true"></iframe>
+
+## Setup
+
+The starting point that I have created for this tutorial, contains a very basic implementation of Tina that includes a blog. It is using our `rich-text` editor and our `TinaMarkdown` component. 
+
+```bash
+# code the repository
+git clone https://github.com/tinacms/examples.git
+
+# Starting commit
+git checkout 7753363ad1e967524359131565f13791d3a378eb
+
+# move into the directory
+cd syntax-highlighting
+
+# install the dependencies 
+yarn
+
+# launch Tina + Next.js
+yarn dev
+```
+
+Once our Next.js application is running, you can navigate to [http://localhost:3000/post/HelloWorld](http://localhost:3000/post/HelloWorld). From there you will see a blog post that has a code block that looks a lot like regular text. When this is within a paragraph it may be hard for you or your editors to tell the difference between the code and regular text. Adding syntax highlighting will easily fix that!
+
+![Image without syntax highlighting](https://res.cloudinary.com/forestry-demo/image/upload/v1642686958/blog-media/syntax-highlighting-with-tina/before-image.webp)
+
+## Adding syntax highlighting
+
+### Installing our package
+
+For this tutorial we are going to use `react-syntax-highlighter` if you prefer a different package, once you make your way through this tutorial and understand how it all works, you can use your favorite package, it will work the same.
```

Comment:
```suggestion
For this tutorial we are going to use `react-syntax-highlighter`. If you prefer a different package, once you make your way through this tutorial and understand how it all works, you can use your favorite package and it will work the same.
```